Dockerfile

```diff
@@ -12,16 +12,19 @@ RUN <<STEPS
   && rm -rf /var/lib/apt/lists /var/cache/apt/archives
 STEPS
 
-ENV RACK_ENV=production
+ARG RACK_ENV="production"
+ARG BUNDLE_WITHOUT="development:quality:test:tools"
+ENV RACK_ENV="$RACK_ENV"
```

Comment:
:axe: Can this be deleted since it's already defined on Line 15 or is there something I'm not seeing that requires this to be reset?

Reply:
ARG will be defined for the docker build, but ARG isn't still set during runtime. This will still be used by bundler when it's run in the environment, correct? So we use ARG to get it settable by the build in the bash script, and also set the environment variable for it's use by bundler during the build and during run.